static/js/mediastream/api.js

```diff
@@ -99,11 +105,29 @@ define(['jquery', 'underscore', 'ua-parser'], function($, _, uaparser) {
 					cb(type, data);
 				}
 				this.send(type, data);
+			}, this),
+			sendEncrypted: _.bind(function(type, data) {
+				if (cb) {
+					cb(type, data);
+				}
+				this.sendEncrypted(type, data);
 			}, this)
 		}
 		return this.apply(name, obj);
 	};
 
+	Api.prototype.sendEncrypted = function(type, data, noqueue) {
```

Comment:
From what I understand, there are other methods that send stuff to the server which the server then needs to read and that doesn't get sent to another user (like the `Hello` message?).